pkg/controller/csr/manager.go

```diff
@@ -45,6 +47,21 @@ func Add(ctx context.Context,
 		Complete(&ReconcileCSR{
 			clientHolder: clientHolder,
 			recorder:     helpers.NewEventRecorder(clientHolder.KubeClient, ControllerName),
+			checks: []func(ctx context.Context, csr *certificatesv1.CertificateSigningRequest) (bool, error){
+				// Case 1: if a CSR comes from a managed cluster that already exists, approve it
```

Comment:
based on the current checks logic, as long as the managedcluster exists, it should be approved even other checks are not passed?